### PR TITLE
Implement node removal

### PR DESCRIPTION
**Pull request by sabaini**

### CODE REVIEW

File: microceph/cmd/microceph/cluster_remove.go [hunk 1]
```diff
@@ -36,15 +41,123 @@ func (c *cmdClusterRemove) Run(cmd *cobra.Command, args []string) error {
...
+	err = client.MClient.DeleteClusterMember(cli, node, force)
+	logger.Debugf("DeleteClusterMember %v: %v", node, err)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
+
+func checkPrerequisites(cli *microCli.Client, name string) (bool, error) {
```
Comment (lmlg) [inline microceph/cmd/microceph/cluster_remove.go:87]: The boolean returned by this function isn't too useful, imo. The returned error already provides the needed information, and checking for both the error _and_ the bool makes things more complex than they should.

**Review by lmlg — CHANGES_REQUESTED** [review]
LGTM, just a small nit.

File: microceph/cmd/microceph/cluster_remove.go (hunk 1 above)
Comment (sabaini) [reply microceph/cmd/microceph/cluster_remove.go]: Ack

**Review by UtkarshBhatthere — APPROVED** [review]
LGTM

**Pull request merged**